build.sbt

```diff
@@ -58,9 +58,9 @@ lazy val plugin: Project =
   (project in file("plugin")) settings (
     sharedSettings: _*
   ) settings (
-    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0-M5" cross CrossVersion.full),
+    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
     libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-compiler" % _ % "provided"),
-    libraryDependencies += "org.scalameta" % "scalahost" % "0.1.0-SNAPSHOT" cross CrossVersion.full,
+    libraryDependencies += "org.scalameta" % "scalahost_2.11.8" % "1.6.0",
```

Comment:
You can use `foo % bar % version cross CrossVersion.full` to have sbt automatically append the full Scala version. To have it respect the binary version, you can do either `foo %% bar % version` or `foo % bar % version cross CrossVersion.Binary`.

But that's fine, I'll change it soon when I release the plugin with your improvements 👍